libs/build-all-ios.sh

```diff
@@ -2,9 +2,10 @@
 
 set -euvx
 
+# This probably needs to be updated, but also includes the toolchain?
 IOS_MIN_SDK_VERSION="11.0"
 # Our short-names for the architectures.
-TARGET_ARCHS=("x86_64" "arm64")
+TARGET_ARCHS=("x86_64" "arm64" "arm64-sim")
```

Comment:
We have some developers using Intel on their machines as well. Is there `x86_64-sim`?

Reply:
x86_64 is the intel sim since there are no x86_64 iOS hardware! Most of the issues stem from the fact that there was only 1 arm64 (iOS hardware), but now there is a sim target (M1) which is what causes most issues in most libs!